db/migrate/20221101182846_add_user_to_like.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ]  kindly reference author instead of user like this ` t.references :author, foreign_key: { to_table: 'users' }`

Reply:
Done!